azure-iot-device/tests/iothub/aio/test_async_clients.py

```diff
@@ -1007,6 +1007,36 @@ def connection_string(self, device_connection_string):
     def sas_token_string(self, device_sas_token_string):
         return device_sas_token_string
 
+    @pytest.fixture
+    def fake_assigned_hub(self):
+        return "DumbledoreArmy.azure-devices.net"
+
+    @pytest.fixture
+    def fake_device_id(self):
+        return "elder_wand"
+
+    @pytest.fixture
+    def symmetric_key(self):
+        return "Zm9vYmFy"
```

Comment:
If this has to be a fixture, this might be better suited to `client_fixtures.py`

Reply:
i had this in the client fixtures but I brought it here as I thought its more specific to Device and DPS. Like the connection string fixture which actually returns the `device_connection_string` ultimately. 

---
In reply to: [355083352](https://github.com/Azure/azure-iot-sdk-python/pull/407#discussion_r355083352) [](ancestors = 355083352)